src/models/transfers.js

```diff
@@ -481,11 +473,27 @@ function * insertTransfers (externalTransfers) {
     converters.convertToInternalTransfer))
 }
 
+function * resendNotification (transferId) {
```

Comment:
I think you want some kind of check here to ensure the party requesting the notification has privileges to views this transfer ... although it doesn't look like the get transfer endpoint has any sort of authentication either.